apps/studio/src/pages/sites/[siteId]/folders/[folderId]/index.tsx

```diff
@@ -42,14 +68,18 @@ const FolderPage: NextPageWithLayout = () => {
     onOpen: onFolderCreateModalOpen,
     onClose: onFolderCreateModalClose,
   } = useDisclosure()
+  const {
+    isOpen: isFolderSettingsModalOpen,
+    onOpen: onFolderSettingsModalOpen,
+    onClose: onFolderSettingsModalClose,
+  } = useDisclosure()
 
   const { folderId, siteId } = useQueryParse(folderPageSchema)
 
-  const [{ title, permalink, children, parentId }] =
-    trpc.folder.readFolder.useSuspenseQuery({
-      siteId,
-      resourceId: folderId,
-    })
+  const [{ title, permalink }] = trpc.folder.readFolder.useSuspenseQuery({
+    siteId: parseInt(siteId),
+    resourceId: parseInt(folderId),
```

Comment:
we should refrain from all the casting and transformations imo. easy to desync or lose reasoning on why the types are the way they are. as discussed offline, since we do not actually do any arithmetic on the resource ids, we technically can keep them in their expected types: siteId: int, resourceId: string. Server should be able to pass them to the query without any transformations too

Reply:
i agree. how do we feel about a PR later on to tidy these types since they are unlikely to change? i want to do a 1 shot change rather than in phases so0 that post change, it's easier to reference and take code